src/folsom_cowboy_app.erl

```diff
@@ -33,19 +33,37 @@
 -define(APP, folsom_cowboy).
 
 start(_Type, _Args) ->
-    Dispatch = cowboy_router:compile(env(dispatch)),
+    Cb = application:ensure_all_started(cowboy),
+    Cbku = application:ensure_all_started(cowboyku),
+
+    case {Cb, Cbku} of
+        {{ok, _}, {error, _}} ->
+            Router = cowboy_router,
+            Cowboy = cowboy;
+        {{error, _}, {ok, _}} ->
+            Router = cowboyku_router,
+            Cowboy = cowboyku;
+        _ ->
+            Cowboy = Router = bleh,
+            exit(no_cowboy_variant)
```

Comment:
We should just go for the default cowboy there I think, especially since Cowboyku is not put as a dependency.